docs/examples/PanelListGroupFill.js

```diff
@@ -0,0 +1,13 @@
+var panelInstance = (
+    <Panel header="Panel heading">
+      Some default panel content here.
+      <ListGroup fill>
+        <ListGroupItem>Item 1</ListGroupItem>
+        <ListGroupItem>Item 2</ListGroupItem>
+        <ListGroupItem>&hellip;</ListGroupItem>
+      </ListGroup>
+      Some more panel content here.
+    </Panel>
+  );
+
+React.render(panelInstance, mountNode);
```

Comment:
Something must have been really out of wack with my local repo. I am seeing the same thing now.